client/network_status.go

```diff
@@ -0,0 +1,154 @@
+package client
+
+import (
+	"fmt"
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/pkg/server"
+	"github.com/skupperproject/skupper/pkg/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (cli *VanClient) NetworkStatus() ([]*types.SiteInfo, error) {
+
+	//Checking if the router has been deployed
+	_, err := cli.KubeClient.AppsV1().Deployments(cli.Namespace).Get(types.TransportDeploymentName, metav1.GetOptions{})
+	if err != nil {
+		return nil, fmt.Errorf("skupper is not installed: %s", err)
+	}
+
+	sites, err := server.GetSiteInfo(cli.Namespace, cli.KubeClient, cli.RestConfig)
+
+	if err != nil {
+		return nil, err
+	}
+
+	versionCheckedSites := cli.checkSiteVersion(sites)
+	siteNameMap := getSiteNameMap(sites)
+
+	services, err := server.GetServiceInfo(cli.Namespace, cli.KubeClient, cli.RestConfig)
+
+	if err != nil {
+		return nil, err
+	}
+
+	var listSites []*types.SiteInfo
+
+	for _, site := range versionCheckedSites {
+
+		if len(site.Namespace) == 0 {
+			return nil, fmt.Errorf("temporarily unable to provide site information")
+		}
+
+		listLinks, err := cli.getSiteLinksStatus(site.Namespace, siteNameMap)
```

Comment:
Suggest that the link status should be for local site only

Reply:
Thanks for finding this, the status of the link is showed only for local site.